docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -3,15 +3,15 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
-= Quarkus - Using OpenID Connect and Keycloak to Centralize Authorizations
```

Comment:
I would leave as it stands today because this is related to `Centralized Authorization`.

Reply:
@pedroigor OK, I wanted to make it crystal clear that it is only related (at the moment) to the bearer tokens, but sure, I can drop this title update, since I'm clarifying it further in the document again...